cmd/notify-nats.go

```diff
@@ -52,7 +53,7 @@ func (n *natsNotify) Validate() error {
 	if !n.Enable {
 		return nil
 	}
-	if _, err := checkURL(n.Address); err != nil {
+	if _, _, err := net.SplitHostPort(n.Address); err != nil {
```

Comment:
> , NATs already supports only this format.

SplitHostPort requires port, doesn't it? it means it can break existing configuration.